docs/python_modules/enchantfilters/SWFFilters.py

```diff
@@ -0,0 +1,30 @@
+from enchant.tokenize import Filter
```

Comment:
What about copyright and license statements?

Reply:
good point, will add!